File: 2493/2493.cpp
class Solution {
public:
    int divideNodesIntoTheMaximumNumbe(TreeNode* root) {
        if(!root) return 0;
        return 1+max(divideNodesIntoTheMaximumNumbe(root->left),divideNodesIntoTheMaximumNumbe(root->right));
    }
};

File: 2493/2493.js
/**
 * @param {TreeNode} root
 * @return {number}
 */
var divideNodesIntoTheMaximumNumbe = function(root) {
    if (!root) return 0;
    return 1 + Math.max(divideNodesIntoTheMaximumNumbe(root.left), divideNodesIntoTheMaximumNumbe(root.right));
};